cost-analyzer/templates/_helpers.tpl

```diff
@@ -91,6 +91,14 @@ Kubecost 2.0 preconditions
     {{- fail "Kubecost no longer includes PodSecurityPolicy by default. Please take steps to preserve your existing PSPs before attempting the installation/upgrade again with the podSecurityPolicy values removed." }}
   {{- end }}
 
+  {{- if ((.Values.kubecostDeployment).leaderFollower).enabled -}}
+    {{- fail "\nIn Kubecost 2.0, Leader Follower architecture has been deprecated, please reach out to support to discuss upgrade paths." -}}
```

Comment:
deprecated ≠ removed. Is it removed (not available with this release) or is it deprecated (available now but being removed in the future)?